code/modules/vehicles/bike.dm

```diff
@@ -96,9 +98,9 @@
 /obj/vehicle/bike/Move(var/turf/destination)
 	if(kickstand) return
 
-
 	//these things like space, not turf. Dragging shouldn't weigh you down.
-	if(istype(destination,/turf/space) || pulledby)
+	var/list/types = typecacheof(list(/turf/space, /turf/simulated/open, /turf/simulated/floor/asteroid))
```

Comment:
Prob. should make this static or make it an object var + initialize it in Initialize/New - only should need to be calculated once.